snap/frontend/components/AccountBalance/index.jsx

```diff
@@ -3,10 +3,10 @@ import { useWalletContext } from '../../context';
 const AccountBalance = () => {
 	const { walletState } = useWalletContext();
 	const { mosaics, currency } = walletState;
-	const { symbol, currencyPerXYM } = currency;
+	const { symbol, price } = currency;
 
 	const xymBalance = mosaics.find(m => 'symbol.xym' === m.name)?.amount || 0;
-	const convertToCurrency = (xymBalance * currencyPerXYM).toFixed(2);
+	const convertToCurrency = (xymBalance * price).toFixed(2);
```

Comment:
On testnet accounts, do we show the price also? 🤔

Reply:
Nope, the price will be disabled to view in testnet.
But for now, I leave it because it's easy for me to test in testnet.